### PR TITLE
Tiny cleanup tweaks to 'remove doc comment node'.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @Hosch250

File: src/Features/Core/Portable/DocumentationComments/CodeFixes/AbstractRemoveDocCommentNodeCodeFixProvider.cs [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.CodeFixes;
-using System.Collections.Generic;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.DiagnosticComments.CodeFixes
 {
-    internal abstract class AbstractRemoveDocCommentNodeCodeFixProvider<TXMLElement> : CodeFixProvider
+    internal abstract class AbstractRemoveDocCommentNodeCodeFixProvider<TXmlElementSyntax> : CodeFixProvider
+        where TXmlElementSyntax : SyntaxNode
```
Comment (Hosch250) [inline src/Features/Core/Portable/DocumentationComments/CodeFixes/AbstractRemoveDocCommentNodeCodeFixProvider.cs]: Whoops, I should have remembered that.  :+1:

File: src/Features/Core/Portable/DocumentationComments/CodeFixes/AbstractRemoveDocCommentNodeCodeFixProvider.cs [hunk 2]
```diff
@@ -29,31 +31,25 @@ internal abstract class AbstractRemoveDocCommentNodeCodeFixProvider<TXMLElement>
             {
                 context.RegisterCodeFix(
                     new MyCodeAction(
-                        c => RemoveDuplicateParamTag(context.Document, context.Span, c)),
+                        c => RemoveDuplicateParamTagAsync(context.Document, context.Span, c)),
```
Comment (Hosch250) [inline src/Features/Core/Portable/DocumentationComments/CodeFixes/AbstractRemoveDocCommentNodeCodeFixProvider.cs]: Whoops, artifact from when I was trying to work out whether to just pass the root and discovered node in and fix it non-async.  :+1:

**Comment by Hosch250** [comment]
Looks great.

**Pull request merged**